qiskit/qobj/utils.py

```diff
@@ -47,6 +48,13 @@ def validate_qobj_against_schema(qobj):
     Raises:
         SchemaValidationError: if the qobj fails schema validation
     """
+    warnings.warn(
+        "The jsonschema validation included in qiskit-terra is "
+        "deprecated and will be removed in a future release. "
+        "If you're relying on this schema validation you should "
+        "pull the schemas from the Qiskit/ibmq-schemas and directly "
+        "validate your payloads with that", DeprecationWarning,
+        stacklevel=2)
```

Comment:
Not a big deal, but this warning is not really necessary since line `qobj._to_dict()` on line 59 will also issue a warning.

Reply:
Yeah, I wasn't sure the best way to handle this so I went with making everything explicit. I wanted to make it clear that both the function and method are deprecated and will be removed in the future. This came up mostly because I was concerned with the `stacklevel` kwarg because without this deprecation warning it would show the `to_dict` line in this function as the source of the deprecated call.

This method should have honestly been deprecated when I removed marshmallow from qobj since all it does after that is call `to_dict(validate=True)`.